Sources/SwiftDocC/Semantics/Snippets/Snippet.swift

```diff
@@ -12,6 +12,27 @@ import Foundation
 public import Markdown
 import SymbolKit
 
+// NOTE: If you update the DocC content, remember to run
+// `swift run generate-symbol-graph` to regenerate the DocC
+// symbol content for the DocC Documentation.
```

Comment:
I don't think a specific directive file is the right place for this information. 

It's also not just about the documentation comment about the public properties of this class (and their documentation). This also applies to all `AutomaticDirectiveConvertible` types (including the introduction of new types).

If anything, maybe this information belongs in a section of the Directives file in the contributor documentation similar to how the Benchmarking file has a section about adding new metrics and there's an entire article about adding feature flags.